### DISCUSSION TITLE
Dockerfile suggestion

### DISCUSSION THREAD

**Discussion by solfrak** [anyproto/contributors#76] [Anytype apps]
Hi, I would really like to become an active developer for this app since I really like the app. Unfortunately I have a lot of trouble setting up my development environment on my machine. I think setting up a Dockerfile or Docker-Compose to setup a dev environment would really help make the first step to a lot of developer into contributing to the project. What do you think?

**Comment by Shampra** [comment]
Look at SamBouwer's work here : https://github.com/SamBouwer/any-docker
